library/src/nvidia_detail/hipblas.cpp

```diff
@@ -4847,8 +4876,29 @@ hipblasStatus_t hipblasZgemvBatched(hipblasHandle_t                   handle,
                                     hipblasDoubleComplex* const       y[],
                                     int                               incy,
                                     int                               batchCount)
+try
 {
+#if CUBLAS_VERSION >= 110700
+    return hipCUBLASStatusToHIPStatus(cublasZgemvBatched((cublasHandle_t)handle,
```

Comment:
Probably should make a future TODO ticket to review moving to a getProcAddress query system so we can cover a few major releases with the same build.   These are for the C API so shouldn't be too tricky.

Reply:
Created [LWPMLSE-523](https://ontrack-internal.amd.com/browse/LWPMLSE-523)